tests/storages_tests/test_with_server.py

```diff
@@ -145,14 +169,16 @@ def test_store_nan_intermediate_values(storage_url: str) -> None:
     assert np.isnan(got_value)
 
 
-def test_multiprocess(storage_url: str) -> None:
+def test_multiprocess(storage_url: str, storage_mode: str) -> None:
     n_workers = 8
     study_name = _STUDY_NAME
-    optuna.create_study(storage=storage_url, study_name=study_name)
+    optuna.create_study(storage=get_storage(storage_url, storage_mode), study_name=study_name)
     with Pool(n_workers) as pool:
-        pool.map(run_optimize, [(study_name, storage_url)] * n_workers)
+        pool.map(run_optimize, [(study_name, storage_url, storage_mode)] * n_workers)
 
-    study = optuna.load_study(study_name=study_name, storage=storage_url)
+    study = optuna.load_study(
+        study_name=study_name, storage=get_storage(storage_url, storage_mode)
+    )
 
     trials = study.trials
     assert len(trials) == n_workers * 20
```

Comment:
You might want to assign a variable to 20 (`num_trials = 20`) so that it is consistent with `run_optimize`.